src/sentry/features/manager.py

```diff
@@ -263,6 +263,34 @@ def batch_has(
                 feature_names, actor, projects=projects, organization=organization
             )
         else:
+            # Fall back to default handler if no entity handler available. We can add batching here
+            # if w
+            project_features = filter(lambda name: name.startswith("projects:"), feature_names)
+            if projects and project_features:
+                results: MutableMapping[str, Mapping[str, bool]] = {}
+                for project in projects:
+                    proj_results = results[f"project:{project.id}"] = {}
+                    for feature_name in project_features:
+                        proj_results[feature_name] = self.has(feature_name, project, actor=actor)
+                return results
+
+            org_features = filter(lambda name: name.startswith("organizations:"), feature_names)
```

Comment:
@wedamija I'm surprised we don't have this fallback logic already somewhere else

Reply:
Yeah, me too. But I checked with others and we couldn't see a path for there to be a fallback here.